package.json

```diff
@@ -5,6 +5,7 @@
   "main": "dist/tester.cjs.js",
   "module": "dist/tester.esm.js",
   "browser": "dist/tester.umd.js",
+  "types": "dist/index.d.ts",
```

Comment:
What's the advantage fo that change ?

Reply:
We're generating types but fields-ant ( and any other place that uses it ) can't find them.

Reply:
( I think )

Reply:
got it, thanks ! 👍